fft/src/roots_of_unity.rs

```diff
@@ -0,0 +1,69 @@
+use lambdaworks_math::field::{
```

Comment:
this was moved out of the TwoAdicity trait and into it's own module inside `fft`.